docs/validate.rst

```diff
@@ -187,70 +187,31 @@ By default, no validation is enforced, but optionally, validation can be enabled
         ...
     ValidationError: "-12" is not a "ipv4"
 
-.. autoclass:: FormatChecker
-    :members:
-    :exclude-members: cls_checks
-
-    .. attribute:: checkers
-
-        A mapping of currently known formats to tuple of functions that validate them and errors that should be caught.
-        New checkers can be added and removed either per-instance or globally for all checkers using the `FormatChecker.checks` decorator.
-
-    .. classmethod:: cls_checks(format, raises=())
-
-        Register a decorated function as *globally* validating a new format.
-
-        Any instance created after this function is called will pick up the supplied checker.
-
-        :argument str format: the format that the decorated function will check
-        :argument Exception raises: the exception(s) raised
-            by the decorated function when an invalid instance is
-            found. The exception object will be accessible as the
-            `jsonschema.exceptions.ValidationError.cause` attribute
-            of the resulting validation error.
 
-        .. deprecated:: v4.14.0
+Some formats require additional dependencies to be installed.
 
-            Use `FormatChecker.checks` on an instance instead.
+The easiest way to ensure you have what is needed is to install ``jsonschema`` using the ``format`` or ``format-nongpl`` extras.
 
+For example:
 
-.. autoexception:: FormatError
-    :noindex:
-    :members:
+    .. code-block:: sh
 
-
-There are a number of default checkers that `FormatChecker`\s know how
-to validate. Their names can be viewed by inspecting the
-`FormatChecker.checkers` attribute. Certain checkers will only be
-available if an appropriate package is available for use. The easiest way to
-ensure you have what is needed is to install ``jsonschema`` using the
-``format`` or ``format_nongpl`` collection of optional dependencies -- e.g.
-
-.. code-block:: sh
-
-   $ pip install jsonschema[format]
-
-which will install all of the below dependencies for all formats.
+        $ pip install jsonschema[format-nongpl]
```

Comment:
Given the narrative before, I think this should be:

```suggestion
        $ pip install jsonschema[format]
```

Also the indentation is off I think. If I interpret it correctly, Sphinx is rendering it as code in a blockquote.